# npm resolution error report

2022-11-04T00:50:31.060Z

While resolving: auth-credentials@0.1.0
Found: next@13.0.1
node_modules/next
  next@"13.0.1" from the root project

Could not resolve dependency:
peer next@"^12.2.5" from next-auth@4.15.0
node_modules/next-auth
  next-auth@"*" from the root project

Fix the upstream dependency conflict, or retry
this command with --force, or --legacy-peer-deps
to accept an incorrect (and potentially broken) dependency resolution.

Raw JSON explanation object:

{
  "code": "ERESOLVE",
  "current": {
    "name": "next",
    "version": "13.0.1",
    "whileInstalling": {
      "name": "auth-credentials",
      "version": "0.1.0",
      "path": "C:\\Users\\Asim Sheikh\\projects\\nextjs\\auth-credentials"
    },
    "location": "node_modules/next",
    "isWorkspace": false,
    "dependents": [
      {
        "type": "prod",
        "name": "next",
        "spec": "13.0.1",
        "from": {
          "location": "C:\\Users\\Asim Sheikh\\projects\\nextjs\\auth-credentials"
        }
      }
    ]
  },
  "currentEdge": {
    "type": "prod",
    "name": "next",
    "spec": "13.0.1",
    "from": {
      "location": "C:\\Users\\Asim Sheikh\\projects\\nextjs\\auth-credentials"
    }
  },
  "edge": {
    "type": "peer",
    "name": "next",
    "spec": "^12.2.5",
    "error": "INVALID",
    "from": {
      "name": "next-auth",
      "version": "4.15.0",
      "whileInstalling": {
        "name": "auth-credentials",
        "version": "0.1.0",
        "path": "C:\\Users\\Asim Sheikh\\projects\\nextjs\\auth-credentials"
      },
      "location": "node_modules/next-auth",
      "isWorkspace": false,
      "dependents": [
        {
          "type": "prod",
          "name": "next-auth",
          "spec": "*",
          "from": {
            "location": "C:\\Users\\Asim Sheikh\\projects\\nextjs\\auth-credentials"
          }
        }
      ]
    }
  },
  "strictPeerDeps": false,
  "force": false
}
